site/content/docs/integrations/prometheus-v2.md

```diff
@@ -43,7 +43,7 @@ Here is an example
 Now restart Prometheus and you should see metrics coming in.
 
 {{< warning >}}
-The Prometheus metrics endpoint has a rate limit of 8 requests per 5 minutes. We recommend using a scrape interval of 60 seconds.
+The Prometheus metrics endpoint has a rate limit of 50 requests per minute. We recommend using a scrape interval equal or above 2 seconds.
```

Comment:
We should mention that the values are cached. Users hitting this endpoint every 2 seconds won't really change much